docs/Tags.md

```diff
@@ -73,19 +81,58 @@ FT.SEARCH myIndex "@cities:{ New York | Los Angeles | Barcelona }"
+FT.SEARCH myIndex "@cities:{ New York } @cities:{Los Angeles} @cities:{ Barcelona }"
+```
+
+## Including punctuation in tags
+
+A tag can include punctuation other than the field's separator (by default, a comma). You do not need to escape punctuation when using the `HSET` command to add the value to a Redis Hash.
+
+For example, given the following index:
+
+```
+FT.CREATE punctuation ON HASH PREFIX 1 test: SCHEMA tags TAG
 ```
 
-## Multi-word tags And escaping
+, you can add tags that contain punctuation like this:
+
+```
+HSET test:1 tags "Andrew's Top 5,Justin's Top 5"
+```
+
+However, when you query for tags that contain punctuation, you must escape that punctuation with a backslash character (`\`).
+
+**NOTE**: In most languages you will need an extra backslash. This is also the case in the redis-cli.
+
+For example, querying for the tag `Andrew's Top 5` in the redis-cli looks like this:
+
+```
+FT.SEARCH punctuation "@tags:{ Andrew\\'s Top 5 }"
+```
 
-Tags can be composed multiple words, or include other punctuation marks other than the field's separator (`,` by default). Punctuation marks in tags should be escaped with a backslash (`\`). *NOTE:* in most languages you will need an extra backslash when formatting the document or query, to signify an actual backslash, so the actual text in redis-cli for example, will be entered as hello\\-world.
 
-It is also recommended (but not mandatory) to escape spaces; The reason is that if a multi-word tag includes stopwords, it will create a syntax error. So tags like "to be or not to be" should be escaped as "to\ be\ or\ not\ to\ be". For good measure, you can escape all spaces within tags.
 
-The following are identical:
+## Tags that contain multiple words
+
+As the examples in this document show, a single tag can include multiple words. We recommend that you escape spaces when querying, though doing so is not required.
+
+You escape tags the same way that you escape punctuation -- by preceding the space with a backslash character (or two backslashes, depending on the programming language and environment).
+
+Thus, you would escape the tag "to be or not to be" like so when querying in the redis-cli:
 
 ```
-@tags:{foo\ bar\ baz | hello\ world}
+FT.SEARCH idx "@tags:{ to\\ be\\ or\\ not\\ to\\ be }"
+```
+
+You should escape spaces because if a tag includes multiple words and some of them are _stop-words_ like "to" or "be," a query that includes these words without escaping spaces will create a syntax error.
 
-@tags:{foo bar baz | hello world }
+You can see what that looks like in the following example:
+
+```
+127.0.0.1:6379> FT.SEARCH idx "@tags:{ to be or not to be }"
+(error) Syntax error at offset 27 near be
 ```
+
+**NOTE:** Stop words are words that are so common that a search engine ignores them. We have a dedicated page about [stop words in RediSearch](https://oss.redislabs.com/redisearch/Stopwords/#default_stop-word_list) if you would like to learn more.
```

Comment:
Readers may not know what stop-words are, so we should give them a short definition and link to our detailed page that includes the canonical default list of stop words.